Gifski/TrimmingAVPlayer.swift

```diff
@@ -210,7 +284,51 @@ final class TrimmingAVPlayerView: AVPlayerView {
 		.toCancellable
 	}
 
-	fileprivate func hideTrimButtons() {
+	var playPauseButtonTarget: AnyObject?
+
+	fileprivate func enableOrDisablePlayButton(enable: Bool) {
+		guard
+			let avTrimView = firstSubview(deep: true, where: { $0.simpleClassName == "AVTrimView" }),
+			let superview = avTrimView.superview
+		else {
+			return
+		}
+		guard let playPauseButton = (superview.subviews
+			.first { $0 != avTrimView }?
+			.subviews
+			.first {
+				guard let button = ($0 as? NSButton),
+				button.action?.description == "playPauseButtonPressed:" else {
+					return false
+				}
+				return true
+			} as? NSButton) else {
+				return
+			}
+		if playPauseButton.target !== self {
+			playPauseButtonTarget = playPauseButton.target
+			playPauseButton.target = self
+		}
+
+		playPauseButton.isEnabled = enable
```

Comment:
This should be a AVPlayerView extension called `trimViewPlayPauseButton`. Then you could simplify `fileprivate var enablePlayButton = true {` to be a getter/setter that proxies to `playPauseButton.isEnabled` directly.

And `playPauseButtonTarget = playPauseButton.target` can be done in `init`.